### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by karanJ2212**
In this milestone following features in redux folder are implemented

 Books actions and reducer  using the [ducks pattern]

Categories actions and reducer using the [ducks pattern]

Configured the Redux Store 

Imported the necessary methods from Redux Toolkit.
Combine both reducers into a root reducer by using configureStore function.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// Action Types
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```
Comment (emmiiorji) [inline src/redux/books/books.js]: You have done great at defining the action types. As specified in the requirements, kindly use the [duck's pattern](https://github.com/erikras/ducks-modular-redux). e.g `REMOVE_BOOK = bookstore/books/REMOVE_BOOK`

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+// Action type for checking the status
+const CHECK_STATUS = 'CHECK_STATUS';
```
Comment (emmiiorji) [inline src/redux/categories/categories.js]: As mentioned above, kindly ensure that all actions follow the [duck's pattern](https://github.com/erikras/ducks-modular-redux).

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @karanJ2212, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

BbcanBbcan3GIF

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by emmiiorji** [comment]
Hi @karanJ2212,

Kindly note that the approval on your dashboard was in error. Sorry about that as this will be reversed as soon as possible. 

Please, implement the changes and request a review again on your dashboard.

**Review by MeqdamAlqudah — COMMENTED** [review]
Hi @karanJ2212 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a pull request that was submitted for another project please create a new pull request and submit your project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by enniopo** [comment]
Hi @karanJ2212, you can now request a re-review for the "Bookstore: add reducers and actions" milestone. Your dashboard has been updated. Thank you, and apologies for the inconvenience.

File: src/redux/books/books.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // Action Types
-const ADD_BOOK = 'ADD_BOOK';
-const REMOVE_BOOK = 'REMOVE_BOOK';
+const ADD_BOOK = 'redux/books/books/ADD_BOOK';
+const REMOVE_BOOK = 'redux/books/books/REMOVE_BOOK';
```
Comment (elisha2kyakpo1) [inline src/redux/books/books.js:3]: - [ ] [OPTIONAL] Just writing `redux/books/ADD_BOOK` or `redux/books/REMOVE_BOOK` it's enough.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @karanJ2212, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did a great job making the changes requested by the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**